### ISSUE TITLE
tidymodels is missing from the setup instructions

### ISSUE THREAD

**Issue by hannesbecher** [carpentries-incubator/high-dimensional-stats-r#58]
In episode 3 - regularised regression, there is a callout (the last one) which requires `tidymodels`. But this package is not part of the setup instructions.

I was going to send a pull request with `tidymodels` added to the file `dependencies.csv` mentioned in the setup code: https://carpentries-incubator.github.io/high-dimensional-stats-r/setup.html but I cannot figure out where this file is located.

**Comment by Alanocallaghan** [comment]
tl;dr: it's fine imo, the tidymodels code is too long to go through in detail, it's just meant to be there to show people that the framework exists. It could be ignored or even removed.

1. The tidymodels code doesn't need to run, so tidymodels doesn't need to be in the setup instructions. I added the tidymodels code because a couple of the curriculum review committees said they felt we should be using tidymodels instead of glmnet directly.
2. dependencies.csv is [here](https://github.com/carpentries-incubator/high-dimensional-stats-r/blob/gh-pages/dependencies.csv) BUT
3. dependencies.csv is made as part of the `make site` recipe [here](https://github.com/carpentries-incubator/high-dimensional-stats-r/blob/gh-pages/bin/list_r_deps.sh) which finds dependencies using [this function](https://github.com/carpentries-incubator/high-dimensional-stats-r/blob/5f70f8b31f496bf7d8eec58de75de8fe36e13c95/bin/dependencies.R#L25)

**Comment by hannesbecher** [comment]
Alright. Thanks v much for the info! I close this issue then.

**Comment by Alanocallaghan** [comment]
Couple was perhaps optimistic but I finally did it https://github.com/carpentries-incubator/high-dimensional-stats-r/wiki
